pyproject.toml

```diff
@@ -49,19 +44,25 @@ markers = [
 ]
 
 [tool.ruff]
-select = ["F", "E", "W", "C90", "N", "UP", "B", "A", "PL", "C4", "EXE", "ISC", "PIE", "T20", "PT", "RET", "SIM", "TCH",
-    "PTH", "PLR", "PLW", "ARG", "ERA", "RUF", "Q"]
+select = ["F", "E", "W", "C90", "N", "UP", "B", "A", "C4", "EXE", "FA", "ISC", "PIE", "T20", "PT", "Q", "RET", "SIM",
```

Comment:
Could you convert this to a sorted list with one line per entry?

Reply:
I don't know if sorting it alphabetically is actually helpful: Right now it is ordered the same way as the rules in [the Ruff docs](https://docs.astral.sh/ruff/rules/) which makes it easier to see changes in new ruff versions